apps/shooting-clubs/acceptance/features/second-contact-name.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+const steps = require('../../');
+
+Feature('Second contact name step');
+
+Before((
+  I,
+  secondContactNamePage
+) => {
+  I.visitPage(secondContactNamePage, steps);
+});
+
+Scenario('The correct form element are present', (
+  I,
+  secondContactNamePage
+) => {
+  I.seeElement('#second-contact-name');
```

Comment:
Thought I did this
http://1.bp.blogspot.com/-FCmeABLxEmM/Tv0gvVx4_eI/AAAAAAAAerY/oBCI4Iem4t8/s1600/facepalm02.png